view/frontend/templates/checkout/onepage/success/paynow_additional_info_note.phtml

```diff
@@ -0,0 +1,5 @@
+<?php if($block->getPaymentType() == 'paynow'): ?>
+<p><strong><i>Note:</i></strong> On your bank app, you will see the payee as <strong>“Omise Payment SG Pte. Ltd“</strong>. Omise is our authorized payment processor to accept PayNow on our behalf.</p>
+<p><strong><i>Supported Banks:</i></strong></p>
+<p style="max-width:420px"><img src="<?php echo $this->getViewFileUrl('Omise_Payment::images/paynow_supportedbanks.png'); ?>" /></p>
+<?php endif; ?>
```

Comment:
Just wonder why we want to separate this additional info note?
Would it possible to merge it to `payment_additional_info.phtml` as it was?

Reply:
Merged back to original file